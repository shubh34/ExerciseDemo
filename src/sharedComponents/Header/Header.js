import React from 'react';
import PropTypes from 'prop-types';

const Header = ({ header, className }) => <h1 className={className}>{header}</h1>;

Header.propTypes = {
	header: PropTypes.string.isRequired,
	className: PropTypes.string,
};
export default Header;
