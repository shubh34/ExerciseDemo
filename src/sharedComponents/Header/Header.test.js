import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('renders without crashing', () => {
	const div = shallow(<Header header='My Blog' />);
	expect(div).toMatchSnapshot();
});
