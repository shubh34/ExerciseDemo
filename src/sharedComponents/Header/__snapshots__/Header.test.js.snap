// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<h1>
  Test
</h1>
`;
